docker/Dockerfile.processor

```diff
@@ -12,6 +12,7 @@ COPY . /app/
 USER app
 
 RUN mkdir -p /tmp/symbols/cache
+RUN mkdir -p /tmp/symbols/tmp
```

Comment:
FYI that comes from here: https://github.com/mozilla-services/socorro/blob/1faba7a9c861cc71f4ff79d5909dbf064d156525/minidump-stackwalk/common.h#L69

It's intended to make sure that you don't put them on separate filesystems because the http fetching code fetches symbol files to a temp path then calls `rename` to move them to the intended name, and that doesn't work across filesystem boundaries. Stupidly, that code doesn't separately check that the directories *exist*, so if a call to `stat` fails (as it would if the directory didn't exist) it prints out the "not on the same filesystem" error. We could fix that to either spit out a more useful error in that case, or just create the directories that are passed in.